routers/api/packages/nuget/links.go

```diff
@@ -30,3 +37,16 @@ func (l *linkBuilder) GetPackageDownloadURL(id, version string) string {
 func (l *linkBuilder) GetPackageMetadataURL(id, version string) string {
 	return fmt.Sprintf("%s/Packages(Id='%s',Version='%s')", l.Base, id, version)
 }
+
+func (l *linkBuilder) GetNextURL() string {
+	u, _ := url.Parse(l.Base)
+	u = u.JoinPath(l.Next.Path)
+	q := u.Query()
+	for k, vs := range l.Next.Query {
+		for _, v := range vs {
+			q.Add(k, v)
+		}
```

Comment:
Not a blocker 
Could you change the `u`, `q`, `vs` shorthand to some significant names or could you annotate them in comments.

Reply:
No, these are the common abbreviations for these types of variables. `u` is often an url, `k` and `v` are `key` and `value`. `vs` `values` in this case. The variables have a very limited scope, so it's easy to look them up.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Sure,